docs/pages/admin-guides/access-controls/resource-rbac.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Per-Resource Role-Based Access Control
+description: Implementing fine-grained access control with RBAC.
+---
+
+Per-resource RBAC enables administrators to control user access on a per-resource basis. Using a granular approach allows you to grant more precise management of user permissions in your environment.
+
+## What is per-resource RBAC?
+
+This capability is particularly useful in complex, multi-tenant environments where user groups have varying levels of access requirements. It allows for precise control over permissions, enabling restricted access to sensitive resources for certain users while granting broader access to others based on their operational needs.
+
+## Use cases
```

Comment:
So far, this guide describes the benefits of per-resource RBAC extensively, including in this use cases section, but we haven't gotten into what per-resource RBAC is yet. 

I think once the reader has a more concrete idea of what per-resourcee RBAC is, we can spend less time talking about potential use cases. I.e., if a reader identifies their own use case, they don't need to read about use cases that don't apply to them.